packages/slate-react/package.json

```diff
@@ -27,6 +27,7 @@
     "slate-plain-serializer": "^0.6.33",
     "slate-prop-types": "^0.5.24",
     "slate-react-placeholder": "^0.1.12",
+    "snapback": "^0.8.0",
```

Comment:
are we using this dependency anywhere?

Reply:
Thanks again! No we are not! That was a library that helped revert the DOM but it turned out to not be sufficient so I wrote the custom Snapshot library.